src/pages/ErrorPage/ErrorPage.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FC } from "react";
+import { useHistory } from "react-router-dom";
+
+interface ErrorPageProps {
+  error?: any;
+}
+
+const ErrorPage: FC<ErrorPageProps> = ({ error }: ErrorPageProps) => {
+  const history = useHistory();
+  const errorText = error ? error.toString() : "Sorry, we can’t find that page.";
+
+  return (
+    <div>
+      <h1>{errorText}</h1>
+      <button onClick={() => history.goBack()}>Back</button>
```

Comment:
I think the content of h1 should always be "Sorry, we can’t find that page." The h1 should have className="mzp-u-title-sm" to be consistent with other page titles.

Then the error message (if there is an error message) can be displayed below the title inside a `p`.

Also, we can use one of the [Mozilla protocol buttons](https://protocol.mozilla.org/patterns/atoms/buttons.html). Neutral or secondary styles look good to me.